README.md

```diff
@@ -497,6 +497,7 @@ Below is an breakdown of all the options.
                 * is_enabled - (Boolean) Enables or disables variance
             * histogram_and_quantiles - Generates a histogram and quantiles
             from the column values
+                * method - (String) Designates preferred method for calculating histogram
```

Comment:
You'll have to list out the possible options.

Reply:
Agreed, should list out the options that are possible.